interfaces/IAirplay.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "Module.h"
+
+namespace WPEFramework {
+    namespace Exchange {
+
+        struct EXTERNAL IAirplay : virtual public Core::IUnknown {
+            enum { ID = ID_AIRPLAY};
+
+            virtual ~IAirplay() { }
+
+            enum state : uint16_t {
+                PLAYING = 0x0001,
+                STOPPED = 0x0002,
+                SUSPENDING = 0x0004
+            };
+
+            struct EXTERNAL INotification : virtual public Core::IUnknown {
+                  enum { ID = ID_AIRPLAY_NOTIFICATION };
+                  virtual void StateChange(const IAirplay::state state) = 0;
+                  virtual void Exit(const uint32_t exitCode) = 0;
+                  virtual void NotifyClient(const string& eventName) = 0;
+            };
+
+            virtual void Register(IAirplay::INotification* airplay) = 0;
+            virtual void Unregister(IAirplay::INotification* airplay) = 0;
+            virtual void FactoryReset() = 0;
+            virtual void SetVisible(bool visibility) = 0;
+            virtual void SystemCommand(const string& command) = 0;
```

Comment:
As a rule of Thumb, we always make COMRPC (and thus interface methods that can pass a Process boundary) return an uint32_t to indicate if, for example, something went wrong during the communication. So preferably suggest to always return an uint32_t, If from the method perspective it should be void, just always return (Core::ERROR_NONE), if something during the communciation fails, the COM layer will change it into a value, indicating Communication error.

Reply:
Thanks, I will use uint32_t for SystemCommand.